Src/Plugins/Tests.Security/TestData/SecurePlaintextSecrets/ExpectedOutputs/SEC101_001.HttpAuthorizationRequestHeader.sarif

```diff
@@ -74,9 +74,9 @@
             }
           ],
           "fingerprints": {
-            "AssetFingerprint/v1": "[host=https://example.com]",
-            "ValidationFingerprint/v1": "[host=https://example.com][secret=SomeAuthorizationKey1111111]",
-            "ValidationFingerprintHash/v1": "d14ab28fe321441f47b1793cbf4c38ca900e94869a5d065fee8ee4b6ecbe67ce"
+            "AssetFingerprint/v1": "[host=example.com][scheme=https]",
```

Comment:
>https [](start = 62, length = 5)

I thought we were going to drop scheme if it is the default of https? what do you think? 

or do you think it is clearer to always have it?